ROADMAP.md

```diff
@@ -0,0 +1,13 @@
+# MDS-Core Roadmap
+
+Work-in-progress.  These are Lacuna priorities; feedback from other MDS-Core contributors is welcome.
+
+* Improved documentation for MDS-Core operation
+* Multi-MDS-version support (MDS 1.0 and later)
+* Add support for Policy Requirements (MDS 1.2)
+* Passenger Services deployment for LADOT (MDS 2.0 alpha)
+* Add support for new modes architecture (MDS 2.0 final)
+* Increased test coverage
```

Comment:
Are there particular packages you have in mind as lacking in test coverage right now? I think most of the service packages in mds-core are all right. Is there a global maximum of line coverage we should be shooting for? I've been on a team once where we went for 95% which struck me as a tad excessive, but 80 feels pretty good.